jwst/resample/resample_step.py

```diff
@@ -35,7 +35,14 @@ class ResampleStep(Step):
         kernel = string(default='square')
         fillval = string(default='INDEF')
         weight_type = option('ivm', 'exptime', default='ivm')
-        pixel_scale_ratio = float(default=1.0) # Ratio of output to input pixel scale
+        outnx = integer(min=1, default=None)
+        outny = integer(min=1, default=None)
```

Comment:
Don't like these names, but I don't have a better idea myself.  `output_size`?

Related, it would be nice if we could pass lists and tuples to the steps.  Theoretically we can do this via:

```
    shape = integer_list(min=list(1, 1), default=None)
```

but I don't know how the command line interface will deal with this.

Reply:
I like this idea but `min=list(1, 1)` does not work (at least not like you would expect it to work: `min`, `max` is used to impose restrictions on the _number of elements_ in the list - not their values).